### PR TITLE
67 read session endpoint

### PR DESCRIPTION
**Pull request by jjrobertson14**
Tested with Postman

### CODE REVIEW

File: server/api.py [hunk 1]
```diff
@@ -286,3 +285,21 @@ def create_session(claims, token_user):
...
+        """ Returns the session with the given id """
+        user = None
+        try:
+            user = token_user
+        except:
+            return malformed_request()      
+
+        # Make sure the session exists
+        try:
+            remote_copy = Session.objects(id=id).first()
```
Comment (joshherkness) [inline server/api.py]: is there any specific reason that we are naming all of these remote_copy?

File: server/api.py [hunk 2]
```diff
@@ -286,3 +285,21 @@ def create_session(claims, token_user):
 
         return jsonify(success="Successfully created session", session=new_session), 200, json_tag
 
+    def read_session(claims, token_user, id):
+        """ Returns the session with the given id """
+        user = None
+        try:
+            user = token_user
```
Comment (AnthonyHewins) [inline server/api.py]: really confused why this is wrapped in a try-except, i don't think it's even possible to get an exception on an assignment

File: server/api.py (hunk 1 above)
Comment (jjrobertson14) [reply server/api.py]: I have no idea, I was just staying consistent with the rest of the file

**Comment by joshherkness** [comment]
@kdjohnson Can we merge this in?

**Review by kdjohnson — APPROVED** [review]

**Pull request merged**